package.json

```diff
@@ -26,6 +27,7 @@
   "devDependencies": {
     "prettier": "^3.0.0",
     "prettier-plugin-organize-imports": "^3.2.2",
+    "run-pty": "^4.0.4",
```

Comment:
this is a sketchy dependency - elsewhere in the repo we use https://www.npmjs.com/package/concurrently to do the same thing - i recommend using it here as well

Reply:
Resolved by 11d276fb